applications/GeoMechanicsApplication/custom_conditions/general_U_Pw_diff_order_condition.cpp

```diff
@@ -16,6 +16,7 @@
 #include "geometries/line_2d_4.h"
 #include "geometries/quadrilateral_3d_4.h"
 #include "geometries/triangle_3d_3.h"
+#include <geometries/line_2d_3.h>
```

Comment:
I would suggest to use double quotes here:
```suggestion
#include "geometries/line_2d_3.h"
```

I expect that then the ordering of the `#include`s becomes more logical.